### PR TITLE
Add documentation and export for eigenvalues

### PR DESCRIPTION
**Pull request by giovannic**
Changes:

 * Pinned squire version to 0.4.8
 * Tidied up github workflow to download latest matching version of chromedriver for chrome
 * Export eigenvalues for covidsim
 * Add documentation for converting between "beta" and "Rt" values

### CODE REVIEW

File: R/export.R [hunk 1]
```diff
@@ -11,22 +11,26 @@ names(countries) <- unique(squire::population$iso3c)
 
 out_dir <- args[1]
 for (iso3c in names(countries)) {
+  R0 <- 3;
```
Comment (richfitz) [inline R/export.R:14]: Sorry, stray semicolon 🙈

File: README.md [hunk 2]
```diff
@@ -91,3 +91,25 @@ Results will be an object representing a table of data. It will have the followi
 Names: The names of the columns of the table. The first name will be t, identifying the time step column. The following names will identify the state and the age group being counted in that column. The age group will be identified in square brackets. E.g. the "S[1]" column represents the count for susceptables in the first age group.
 
 Y: A 2D array representing the rows of the table. The first dimension will depend on the time steps and resolution of the model. The second dimension will be the same size as "Names".
+
+#### beta and Rt
+
+The model is parameterised with a country specific `beta` value.
+
+To translate Rt values into beta values, you can divide them by the country
```
Comment (richfitz) [inline README.md:99]: Way out of my lane here, but in sircovid discussions about beta vs Rt there was some issues about accounting for depletion of susceptibles - is that somehow accounted for here?

File: R/export.R [hunk 3]
```diff
@@ -11,22 +11,26 @@ names(countries) <- unique(squire::population$iso3c)
...
+    ICU_bed_capacity = 1,  #dummy value
+    R0 = R0
   )
 
   country_data <- list(
     population = pars$population,
     contactMatrix = pars$mix_mat_set,
-    beta = pars$beta
+    beta = pars$beta,
+    eigen = R0 / pars$beta
```
Comment (richfitz) [inline R/export.R:26]: Would we ever want the eigenvectors too? If so consider renaming this to eigenvalue to be clearer

File: README.md (hunk 2 above)
Comment (giovannic) [reply README.md:99]: I've let OJ know, he'll update beta estimates upstream if it's relevant.

**Pull request merged**